### PR TITLE
changes needed for SNMP docker upgrade to stretch build

### PR DESCRIPTION
**Pull request by samaity**
Signed-off-by: Sangita Maity [samaity@linkedin.com](mailto:samaity@linkedin.com)

**- What I did**

I was getting lots of import error during SNMP docker upgrade to stretch build. Made changes accordingly to fix those issues. May be python compatibility issue as by default it is using python3 and there's no exceptions module in python3

**- How I did it**
Made it compatible with python3 whenever required.

**- How to verify it**
Use this PR with snmp docker upgrade PR in sonic-buildimage.

### CODE REVIEW

File: sonic_eeprom/eeprom_base.py [hunk 1]
```diff
@@ -13,6 +13,9 @@
 
 try:
     import exceptions
+except ImportError:
+    import builtins as exceptions
```
Comment (jleveque) [inline sonic_eeprom/eeprom_base.py]: Suggest to add comments here for posterity's sake:

```
try:
    import exceptions              # Python 2
except ImportError:
    import builtins as exceptions  # Python 3
```

File: sonic_eeprom/eeprom_dts.py [hunk 2]
```diff
@@ -2,8 +2,11 @@
 # Copyright 2012 Cumulus Networks LLC, all rights reserved
 
 try:
-    import os
     import exceptions
+except ImportError:
+    import builtins as exceptions
```
Comment (jleveque) [inline sonic_eeprom/eeprom_dts.py]: Same as above

File: sonic_sfp/sff8436.py [hunk 3]
```diff
@@ -16,7 +16,7 @@
     import types
     from math import log10
     from sff8024 import type_of_transceiver
-    from sffbase import sffbase
+    from .sffbase import sffbase
```
Comment (jleveque) [inline sonic_sfp/sff8436.py]: Why are these dots necessary now? What was the error you were seeing?

Comment (samaity) [reply sonic_sfp/sff8436.py]: I was getting import error like **module not found**.

File: sonic_eeprom/eeprom_base.py (hunk 1 above)
Comment (samaity) [reply sonic_eeprom/eeprom_base.py]: Added comments for all the modifications behind both python 2 and python 3 support

File: sonic_sfp/sff8436.py (hunk 3 above)
Comment (samaity) [reply sonic_sfp/sff8436.py]: Dot module supports both Python 2 and Python 3 using explicit relative import methods. Added comments wherever needed

**Comment by lguohan** [comment]
please resolve conflict as well.

**Pull request merged**